services/github/github-search.tester.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const Joi = require('joi')
+const { isMetric } = require('../test-validators')
+
+const t = (module.exports = require('../create-service-tester')())
+
+t.create('hit counter')
+  .get('/torvalds/linux/goto.json')
```

Comment:
This isn't related to the refactored tests in this PR but the GH Search hit counter badge isn't rendering for me in the staging deploy env.  You mentioned in https://github.com/badges/shields/issues/1358#issuecomment-448743020 that perhaps it is just that particular repo that is slow, so would it be worth using a different repo for this test?

Reply:
Yea, I think it would. Perhaps we should also update the example, which is what renders in the staging deploy.

One of the open issues PRs also tends to time out. I think it's the undefined label test. We could try it on a smaller repo or give it a longer timeout.

I try to avoid changing things while they're being moved because it's hard to compare what's changed, so let's handle this in a follow-on.